be/src/runtime/exec_env_init.cpp

```diff
@@ -286,8 +368,8 @@ Status ExecEnv::_init_mem_env() {
     while (!is_percent && pk_storage_page_cache_limit > MemInfo::mem_limit() / 2) {
         pk_storage_page_cache_limit = storage_cache_limit / 2;
     }
-    StoragePageCache::create_global_cache(storage_cache_limit, index_percentage,
-                                          pk_storage_page_cache_limit, num_shards);
+    _storage_page_cache = StoragePageCache::create_global_cache(
+            storage_cache_limit, index_percentage, pk_storage_page_cache_limit, num_shards);
```

Comment:
Why generate a raw pointer to return ExecEnv
Instead of `make_unique` here a `StoragePageCache` unique_ptr

Reply:
_storage_page_cache is decleared in exec_env.h, 
 unique_ptr needs complete type, which means we need to #include "StoragePageCache", will make pre-compile slower.

Reply:
> Why generate a raw pointer to return ExecEnv Instead of `make_unique` here a `StoragePageCache` unique_ptr

a complete refactor should add a exec_env_internal, which uses unique_ptr instead of raw ptr.